docs/api-v1.md

```diff
@@ -592,7 +596,47 @@ Base endpoint is: `/ocs/v2.php/apps/spreed/api/v1`
 * `moderator_demoted` - {actor} demoted {user} from moderator
 * `guest_moderator_promoted` - {actor} promoted {user} to moderator
 * `guest_moderator_demoted` - {actor} demoted {user} from moderator
-        
+
+## Search
+
+### Search chat messages
+
+The search uses the server endpoint, so it is not available for guest users.
+
+* Method: `GET`
+* Endpoint: `/index.php/core/search` (from the root, it does not use the base OCS endpoint)
```

Comment:
well this sucks, means it can't be used by the mobile clients :crying_cat_face:

Reply:
Well, I just used what is provided by the server. Anyway, I guess that we could provide our own endpoint in Talk that just calls the standard search internally.